client/prisma/schema.prisma

```diff
@@ -0,0 +1,200 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model Keywords {
+  id        String        @id @default(cuid()) // prisma generates unique id at runtime not implemented database level
+  name      String        @unique @db.VarChar(255)
+  createdAt DateTime      @default(now())
+  updatedAt DateTime      @updatedAt
+  deletedAt DateTime? //optional field
+  users     UserKeyword[]
+  jobs      JobKeyword[]
+  keywords  GigKeyword[]
+}
+
+model Users {
+  id              String            @id @default(cuid())
+  avatarUrl       String //suggestion: making this field optional
+  coverPhotoUrl   String //suggestion: making this field optional
+  nounce          String // looks like authentication is planned to use a wallet extension other than the tradional email/pass
+  // as per https://github.com/Developer-DAO/DAO-job-board/issues/4
+  // so we need the above nounce field
+  // reference article : https://www.toptal.com/ethereum/one-click-login-flows-a-metamask-tutorial
+  // reference repo:
+  email           String?           @db.VarChar //made optional because authentication will be through a wallet extension
+  username        String            @unique @db.VarChar
+  preferences     String            @unique @db.VarChar //suggestion: making this field optional and not unique
+  walletAddress   String            @unique @db.VarChar(128)
+  disabledAt      DateTime? // optional field
+  createdAt       DateTime          @default(now())
+  updatedAt       DateTime          @updatedAt
+  deletedAt       DateTime? //optional field
+  links           Links[] //this field is only used for querying relation not implemented in database level
+  companyOwned    Companies[] //this field is only used for querying relation not implemented in database level
+  jobs            Jobs[] //this field is only used for querying relation not implemented in database level
+  gigs            Gigs[] //this field is only used for querying relation not implemented in database level
+  keyWords        UserKeyword[] //this field is only used for querying relation not implemented in database level
+  companies       CompanyMembers[] //this field is only used for querying relation not implemented in database level
+  JobApplications JobApplications[] //this field is only used for querying relation not implemented in database level
+  GigApplications GigApplications[]
```

Comment:
Change `JobApplications` to use camelCasing -> `jobApplications`

Change `GigApplications` to use camelCasing -> `gigApplications`